### PR TITLE
Visual config editor

### LINKED ISSUE #12: Create a graphical configuration editor
At the moment, configuration editing is done through a YAML based editor window, which provides advanced control, but at the cost of usability.

At some point, a proper graphical editor should be implemented, but not until the exact contents of the configuration file are unlikely to change.

### PR DESCRIPTION
**Pull request by WilliamsJack**
We're getting _really_ fancy now.

This PR implements a **VISUAL CONFIG EDITOR** featuring:

- A **VISUAL CONFIG EDITOR**
- Switching between the **VISUAL CONFIG EDITOR** and the so-called "aDVaNcED eDItOr" (but I mean, really, are you going to use that when you could use the **VISUAL CONFIG EDITOR**?)
- Fixes to [json-editor](https://github.com/SFXRescue/json-editor) that make it, in our (can I say humble?) opinion, even more epic and worthy of our **VISUAL CONFIG EDITOR**
- An alert/notification system so you know the saved/loaded status of the configs in your **VISUAL CONFIG EDITOR**
- A single schema that can be used to validate all S.A.R.T. configs (not only in the **VISUAL CONFIG EDITOR**)
- Did I mention the **VISUAL CONFIG EDITOR**?

**VISUAL CONFIG EDITOR** closes #12

### CODE REVIEW

File: css/sart.style.css [hunk 1]
```diff
@@ -16,22 +16,17 @@
   }
 }
 
-table {
+#thermal_camera table {
```
Comment (Conr86) [inline css/sart.style.css]: What do you mean I shouldn't blindly apply a very specific use case style to every single element of that type??

File: index.html [hunk 2]
```diff
@@ -40,6 +40,9 @@
 	<!-- Font Awesome -->
 	<script defer src="js/fontawesome/fontawesome.js"></script>
 	<script defer src="js/fontawesome/solid.js"></script>
+	<!-- JSON Editor -->
+	<script src = "js/jsoneditor.js"></script>
+	<script src = "js/config.schema.js"></script>
```
Comment (Conr86) [inline index.html]: Should we rename this to sart.config.schema.js?

File: css/sart.style.css (hunk 1 above)
Comment (WilliamsJack) [reply css/sart.style.css]:
```
* {
    background-color: yellow;
}
```

File: index.html (hunk 2 above)
Comment (WilliamsJack) [reply index.html]: I think that's a good idea, also keeps it consistent.

**Comment by Conr86** [comment]
Brilliant. This has become one beautiful interface.

**Pull request merged**